dev-tools/scripts/README.md

```diff
@@ -159,3 +159,24 @@ and prints a regular expression that will match all of them
 
 TBD
 
+
+### cherrypick.sh
+
+    Usage: dev-tools/scripts/cherrypick.sh [<options>] <commit-hash> [<commit-hash>...]
+     -b <branch> Sets the branch to cherry-pick to, typically branch_Nx or branch_x_y
+     -s          Skips precommit test. WARNING: Always run precommit for code- and doc changes
+     -t          Run the full test suite during check, not only precommit
+     -n          Skips git pull of target branch. Useful if you are without internet access
+     -a          Enters automated mode. Aborts cherry-pick and exits on error
+     -r <remote> Specify remote to push to. Defaults to if other than 'origin'
+     -p          Push to remote. Only done if both cherry-pick and tests succeeded
+     WARNING: Never push changes to a remote branch before a thorough local test
+    
+    Simple script for aiding in back-porting one more (trivial) commits to other branches.
+    On merge conflict the script will run 'git mergetool'. See 'git mergetool --help'
+    for help on configuring your favourite merge tool. Check out Sublime Merge (smerge).
+    
+    Example:
+      # Backport two commits to both stable and release branch
+      dev-tools/scripts/cherrypick.sh -b branch_9x  deadbeef0000 cafebabe1111
```

Comment:
can we allow specifying multiple `-b` options?

Reply:
Thought of it, guess it would be possible

Reply:
It turned out to be not too hard, test it out